src/xml.tsx

```diff
@@ -174,6 +174,7 @@ export class SvgFromXml extends Component<XmlProps, XmlState> {
     try {
       this.setState({ ast: xml ? parse(xml) : null });
     } catch (e) {
+      this.props.onError?.(e);
```

Comment:
Nice catch! Shouldn't we use the same logic as introduced in https://github.com/software-mansion/react-native-svg/commit/3c32a6fdd580dea53ecc271288155117ce040b55, so we use `console.error(e);` as the default value of `onError` if other is not provided?

Reply:
`console.error(e);` does not necessarily need to operate by default.

```suggestion
    this.props.onError ? this.props.onError(e) : console.error(e);
```

Is it correct to say this way? That seems like a good approach!